protocols/delegate/contracts/Delegate.sol

```diff
@@ -18,9 +18,12 @@ pragma solidity 0.5.12;
 pragma experimental ABIEncoderV2;
 
 import "@airswap/delegate/contracts/interfaces/IDelegate.sol";
+import "@airswap/indexer/contracts/interfaces/IIndexer.sol";
 import "@airswap/swap/contracts/interfaces/ISwap.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
```

Comment:
Don't you need to import Types?

Reply:
Good question. It's actually imported in IDelegate.sol